src/plugins/mergeCells/cellsCollection.js

```diff
@@ -41,7 +41,7 @@ class MergedCellsCollection {
    * @return {String}
    */
   static IS_OVERLAPPING_WARNING(newMergedCell) {
-    return toSingleLine`The merged cell declared at [${newMergedCell.row}, ${newMergedCell.col}] overlaps with the other declared merged 
+    return toSingleLine`The merged cell declared at [${newMergedCell.row}, ${newMergedCell.col}] overlaps with the other declared merged
```

Comment:
The space between "merged" and "cell" shouldn't be removed, after this change the warning contains merged words.

`The merged cell declared at [29, 1] overlaps with the other declared mergedcell. The overlapping merged cell was not added to the table, please fix your setup.`